lib/backpex/fields/upload.ex

```diff
@@ -55,14 +55,15 @@ defmodule Backpex.Fields.Upload do
   **Parameters**
   * `:socket` - The socket.
   * `:item` (struct) - The item without its changes.
+  * `:change` (map) - The current change / attrs that will be passed to the changeset function.
```

Comment:
I think we should use only one word here to reduce confusion between change / attrs / params (mentioned elsewhere).

- Are these the form params coming from the user? Then we should call it `params`.
- Are these the changes after casting from the Ecto changeset? Then we should call it `changes` (plural).

https://hexdocs.pm/ecto/Ecto.Changeset.html#module-the-ecto-changeset-struct

Do we have the `changes` available at this point? I think, that would be best (in case we have a choice).

Reply:
These are the form params. I renamed it to `params`. The form is called `change` in Backpex so this is a little bit confusing. Maybe we should refactor it in Backpex in the future (see [`form_component.ex#L84`](https://github.com/naymspace/backpex/blob/0.4.0/lib/backpex/live_components/form_component.ex#L84))